taxonomy/utils.py

```diff
@@ -662,3 +663,31 @@ def duplicate_xblock_skills(source_xblock_uuid, xblock_uuid):
     for source_xblock_skill in source_xblock_skills:
         source_xblock_skill.xblock = xblock
         duplicate_model_instance(source_xblock_skill)
+
+
+def update_xblock_skills_verification_counts(usage_key: str, verified_skills: List[int], ignored_skills: List[int]):
+    """
+    Update xblock skill verified and ignored count.
+
+    Arguments:
+        usage_key (str): usage_key of xblock.
+        verified_skills (List[int]): list of verified skill ids.
+        ignored_skills (List[int]): list of ignored skill ids.
+    """
+    xblock_model, identifier = get_product_skill_model_and_identifier(ProductTypes.XBlock)
+    xblock = xblock_model.objects.filter(**{identifier: usage_key}).first()
```

Comment:
Why not just use `objects.get` instead of `objects.filter` if we're expecting a single xblock?

Reply:
@pkulkark Get will raise an error if xblock is not found. This way we can check and log an error and return (missed to add return statement, updated!).

Reply:
@navinkarkera We could put it in a try-catch block and log the required error in the catch block and return. I'm not completely opposed to using `filter` but it just feels a bit unconventional.

Reply:
@pkulkark The filter option avoids try/catch boilerplate, but I agree that using `get` makes more sense here as we want only one object. Updated :+1: